code/game/machinery/camera/camera.dm

```diff
@@ -41,6 +41,10 @@ var/list/camera_names=list()
 	var/vision_flags = SEE_SELF //Only applies when viewing the camera through a console.
 	var/health = CAMERA_MAX_HEALTH
 
+/obj/machinery/camera/initialize()
```

Comment:
Where does this method get called again?

Reply:
When the game starts, it loops through all atoms in the world and calls their `initialize()`

Reply:
Never mind I found it: https://github.com/vgstation-coders/vgstation13/blob/Bleeding-Edge/code/controllers/subsystem/objects.dm

Reply:
roundstart